Ports/abseil/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=abseil
+useconfigure=true
+version=20230802.0
+depends=(  )
+configopts=(
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    "-DABSL_PROPAGATE_CXX_STD=ON"
+    "-DABSL_ENABLE_INSTALL=ON"
+    "-DABSL_BUILD_TESTING=OFF"
+)
+files=(
+    "git+https://github.com/abseil/abseil-cpp.git#${version}"
+)
+workdir="abseil-cpp.git"
```

Comment:
Does this work without that? It should, as far as I am aware.

EDIT: Wait, no, I broke that. I'll try and fix it in the afternoon. :yakplus:

Reply:
The path of abseil needs to be predictable from other ports (protobuf) so that the CMake files in this directory can be found.

Reply:
Not resolved.